server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessageContentExtractor.java

```diff
@@ -67,12 +91,8 @@ private MessageContent parseMultipartMixed(Multipart multipart) throws IOExcepti
         List<Entity> parts = multipart.getBodyParts();
         if (! parts.isEmpty()) {
             Entity firstPart = parts.get(0);
```

Comment:
From above code, the display is affected by Mime ordering. With the solution I suggest, it is not. Making the behaviour more consistent. That is why I propose this modification.